configuration_script.sh

```diff
@@ -0,0 +1,50 @@
+setup_source_cluster() {
+    #This function creates a new project and application for migration on source cluster.
+    oc login -u opentlc-mgr -p 'r3dh4t1!' https://master.rdj2ocp3.mg.dog8code.com:443 --insecure-skip-tls-verify
```

Comment:
@nachandr , This should be sanitized.

```suggestion
    oc login -u USERNAME -p 'PASSWORD' https://master.your-ocp3-cluster.mg.dog8code.com:443 --insecure-skip-tls-verify
```